compiler/luci/lang/include/luci/IR/Nodes/CircleTransposeConv.h

```diff
@@ -47,6 +47,11 @@ class CircleTransposeConv final
   loco::Node *outBackprop(void) const { return at(2)->node(); }
   void outBackprop(Node *node) { at(2)->node(node); }
 
+  /**
+   * @note  "bias" of TransposeConv is only supported by circle node, not by tflite.
```

Comment:
`bias` of TransposeConv is supported in tf v2.3.0 while it is an optional input.
(https://github.com/tensorflow/tensorflow/blob/v2.3.0/tensorflow/lite/kernels/transpose_conv.cc#L273-L274)

It would be better to remove this line.

Reply:
I've found [this](https://github.com/tensorflow/tensorflow/commit/43b8f6e71001be7bacee4559e3021965ae67cf33). Seems like TConv on tflite support optional 4th argument as bias, just like circle IR.

Will amend this line :)